episodes/collaboration.md

```diff
@@ -6,96 +6,53 @@ exercises: 2
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using R Markdown and `{sandpaper}`?
+- Why is effective collaboration important for lesson development?
+- When, if ever, is it best to establish structured project governance?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with the new lesson template
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+By the end of this session, participants should be able to...
 
-::::::::::::::::::::::::::::::::::::::::::::::::
-
-## Introduction
-
-This is a lesson created via The Carpentries Workbench. It is written in
-[Pandoc-flavored Markdown][pandoc] for static files (with extension `.md`) and
-[R Markdown][r-markdown] for dynamic files that can render code into output
-(with extension `.Rmd`). Please refer to the [Introduction to The Carpentries
-Workbench][carpentries-workbench] for full documentation.
-
-What you need to know is that there are three sections required for a valid
-Carpentries lesson template:
-
- 1. `questions` are displayed at the beginning of the episode to prime the
-    learner for the content.
- 2. `objectives` are the learning objectives for an episode displayed with
-    the questions.
- 3. `keypoints` are displayed at the end of the episode to reinforce the
-    objectives.
-
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
-
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
+- recommend tools and practices lesson developers can adopt to aid collaboration.
+- draw on their own experience to advise lesson developers on how to collaborate effectively.
 
-::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
-
-::::::::::::::::::::::::::::::::::::: challenge 
-
-## Challenge 1: Can you do it?
-
-What is the output of this command?
-
-```r
-paste("This", "new", "lesson", "looks", "good")
-```
+::::::::::::::::::::::::::::::::::::::::::::::::
 
-:::::::::::::::::::::::: solution 
 
-## Output
- 
-```output
-[1] "This new lesson looks good"
-```
+:::::::::::::::::::::::::::::::::::::::::::::::::: checklist
 
-:::::::::::::::::::::::::::::::::
+## Reading List
 
+- FIXME
 
-## Challenge 2: how do you nest solutions within challenge blocks?
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-:::::::::::::::::::::::: solution 
 
-You can add a line with at least three colons and a `solution` tag.
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-:::::::::::::::::::::::::::::::::
-::::::::::::::::::::::::::::::::::::::::::::::::
+## Discussion Questions
 
-## Figures
+- What decision-making framework(s) have you used in your previous collaborations?
```

Comment:
Might broaden this a bit as folks may not have experience with any specific decision-making frameworks. 
```suggestion
- How have you made decisions with collaborators in the past?
- Did you use any formal decision-making framework(s)?
- What might be the advantages and drawbacks of using a formal decision-making framework?
```
Also added a pro/con discussion question - sorry for combining suggestions

Reply:
Maybe a discussion of working alone vs collaboration pros and cons would be a good addition based on the objectives for this lesson.